create_buildpack.go

```diff
@@ -0,0 +1,126 @@
+package pack
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/BurntSushi/toml"
+	"github.com/pkg/errors"
+
+	"github.com/buildpacks/pack/internal/build"
+	"github.com/buildpacks/pack/internal/dist"
+	"github.com/buildpacks/pack/internal/style"
+)
+
+var (
+	BuildpackLanguages = map[string]interface{}{
+		"bash":   createBashBuildpack,
+		"go":     createGolangBuildpack,
+		"golang": createGolangBuildpack,
+	}
+
+	bashBinBuild = `
+#!/usr/bin/env bash
+
+set -eu
```

Comment:
Can we default to `set -euo pipefail`? I've seen some funky bugs in buildpacks when folks forget pipefail for curl requests, and it would be a nice default :)

Reply:
yup

Reply:
fixed in a849bf6